.circleci/config.yml

```diff
@@ -1354,6 +1354,44 @@ workflows:
       - aws-summary:
           context: Slack
 
+  nightly-comprehensive-functional-regression:
+    triggers:
+      - schedule:
+          cron: "15 4 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - fast-build-artifact:
+          context: Slack
+          workflow-name: freeze restart
+      - start-singlejob-testnet:
+          context: Slack
+          requires:
+            - fast-build-artifact
+          workflow-name: "nightly-comprehensive-functional-regression"
+          infra-branch: services-with-platform070
```

Comment:
Why do we need this infra branch? Shouldn't it be the default `hashgraph-hedera-services`?

Reply:
Done, the default  is used. Thanks for the good catch.